linera-service/src/proxy/main.rs

```diff
@@ -299,6 +300,15 @@ where
             GenesisConfigHashQuery => Ok(Some(RpcMessage::GenesisConfigHashResponse(Box::new(
                 self.genesis_config.hash(),
             )))),
+            UploadBlobContent(content) => {
+                let blob = Blob::new(*content);
+                let id = blob.id();
+                ensure!(
+                    self.storage.maybe_write_blobs(&[blob]).await?[0],
+                    "Blob not found"
```

Comment:
This ends up being a panic?

Reply:
It returns an `Err`. I think the proxy will just return an error to the request then (otherwise the other `?`s in this function wouldn't make sense).

Reply:
I missed that this function returns `anyhow::Result`.